src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -71,7 +71,7 @@
     private static final String CONTENT_TYPE_FORM_URLENCODED = "application/x-www-form-urlencoded; charset=utf-8";
     private static final Pattern CHARSET_PATTERN = Pattern.compile("(?i)\\bcharset=\\s*\"?([^\\s;\"]*)");
     private static final Pattern CONTENT_DISPOSITION_FILENAME_PATTERN =
-            Pattern.compile("(attachment|inline);\\s*filename\\s*=\\s*\"([^\"]*)\"");
+            Pattern.compile("(attachment|inline|form-data);.*filename\\s*=\\s*(?<unquoted>\"(?<quoted>[^\"]*)\"|[\\w.-]+)");
```

Comment:
Is it wise to read it from a config setting?
It looks to me that very soon we will find yet another website which delivers other stuff.

Reply:
this should be a catch all approach, if we find stuff this doesn't match we need to add it to the test and expand the pattern

Reply:
A config value can also only changed via a patch and we probably do not want this to differ from product to product